app/presenters/flow_registration_presenter.rb

```diff
@@ -88,10 +88,6 @@ def flows_content
       end
   end
 
-  def state
```

Comment:
I wonder if it was related to this: https://github.com/alphagov/smart-answers/blob/master/doc/smart-answer-flow-development/archiving.md.

Perhaps we can remove this doc as well (and the link to it in the README)? Looks like it's pretty out-of-date.

Reply:
Sure - I'll see about doing that.

Reply:
I'll plan to do that after this PR actually since it's not interconnected.